### PR TITLE
Prerelease update source default for prerelease builds

### PR DESCRIPTION
**Pull request by gabriel**
Enables the updater for prerelease builds. Will enable real updates once we test a bunch with this flow.

Also fix build number to be current date since last commit date isn't always increasing.

### CODE REVIEW

File: go/client/cmd_update.go [hunk 1]
```diff
@@ -34,8 +34,10 @@ func NewCmdUpdate(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comman
 				Usage: "Destination of where to apply update.",
 			},
 			cli.StringFlag{
-				Name:  "s, source",
-				Usage: fmt.Sprintf("Update source (%s).", sources.Sources),
+				Name: "s, source",
+				Usage: fmt.Sprintf("Update source (%s). Default is \"%s\".",
```
Comment (maxtaco) [inline go/client/cmd_update.go]: use `%q` to mean `\"%s\"`

File: go/install/sources/prerelease.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build prerelease
```
Comment (maxtaco) [inline go/install/sources/prerelease.go]: what does this tag mean?  nightly?

**Comment by maxtaco** [comment]
Ok, looks good!

File: go/install/sources/prerelease.go (hunk 2 above)
Comment (gabriel) [inline go/install/sources/prerelease.go]: yeah prereleases can happen more often than nightly so calling it prerelease...  but yeah these are the same builds as nightly

**Pull request merged**